src/Payone/Webhook/MessageBus/MessageHandler/NotificationForwardHandler.php

```diff
@@ -4,21 +4,23 @@
 
 namespace PayonePayment\Payone\Webhook\MessageBus\MessageHandler;
 
+use Monolog\Level;
+use Monolog\Logger;
 use PayonePayment\DataAbstractionLayer\Entity\NotificationForward\PayonePaymentNotificationForwardEntity;
 use PayonePayment\DataAbstractionLayer\Entity\NotificationTarget\PayonePaymentNotificationTargetEntity;
 use PayonePayment\Payone\Webhook\MessageBus\Command\NotificationForwardCommand;
-use Psr\Log\LoggerInterface;
 use Shopware\Core\Framework\Context;
 use Shopware\Core\Framework\DataAbstractionLayer\EntityRepository;
 use Shopware\Core\Framework\DataAbstractionLayer\Search\Criteria;
 use Shopware\Core\Framework\DataAbstractionLayer\Search\EntitySearchResult;
+use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\Messenger\Handler\MessageSubscriberInterface;
 
 class NotificationForwardHandler implements MessageSubscriberInterface
 {
     public function __construct(
         private readonly EntityRepository $notificationForwardRepository,
-        private readonly LoggerInterface $logger
+        private readonly Logger  $logger
```

Comment:
why did you remove the LoggerInterface?

i think the loggerinterface is better, because some other devs could replace the logger with another logger, if they want. In your case, the constructor will fail, if another logger will be passed.

Reply:
I need to understand the log status with the `Monolog\Level` in response status.

The Monolog\Logger, with its `addRecord()` method, allows us to dynamically utilize status levels such as `info, error, critical, and warning`.

Using `Psr\Log\LoggerInterface` doesn't seem to achieve the same. In fact, I observed that the Shopware Core Framework and many Shopware plugins use `Monolog`. I assume that this should not pose any issues. I just wanted to ensure that I don't have to write additional code if there is a Shopware package available.

Reply:
ok make sense for me. Thank for clarification